pkg/loader/compose/v1v2.go

```diff
@@ -268,6 +269,50 @@ func libComposeToKomposeMapping(composeObject *project.Project) (kobject.Kompose
 			log.Infof("Service name in docker-compose has been changed from %q to %q", name, normalizeServiceNames(name))
 		}
 	}
+	//Handle Volumes
+	for name, _ := range komposeObject.ServiceConfigs {
+		vols := handleVolume(name, komposeObject)
+		var FinalVol []kobject.Volumes
+		for _, v := range vols {
+			if !findDuplicateVol(v, FinalVol) {
+				FinalVol = append(FinalVol, v)
+			}
+		}
+		var count int
+		for i, _ := range FinalVol {
+			vol := &FinalVol[i]
+			var err error
+			vol.VolumeName, vol.Host, vol.Container, vol.Mode, err = transformer.ParseVolume(vol.MountPath)
+			if err != nil {
+				log.Warningf("Failed to configure container volume: %v", err)
+				continue
+			}
+			log.Debug("Volume name %s", vol.VolumeName)
+
+			if vol.VFrom == "" {
+				vol.PVCName = fmt.Sprintf("%s-claim%d", vol.SvcName, count)
+				count++
+			}
+
+		}
+		var temp = komposeObject.ServiceConfigs[name]
+		temp.Vols = FinalVol
+		komposeObject.ServiceConfigs[name] = temp
+	}
+
+	for _, service := range komposeObject.ServiceConfigs {
+		for v, _ := range service.Vols {
+			vol := &service.Vols[v]
+			if vol.VFrom != "" {
+				for _, j := range komposeObject.ServiceConfigs[vol.VFrom].Vols {
+					if vol.MountPath == j.MountPath {
+						vol.PVCName = j.PVCName
+					}
+				}
+			}
+		}
+	}
+
```

Comment:
Is it possible to refactor this into it's own function (parseV1V2Volumes maybe?)